package/MDAnalysis/coordinates/base.py

```diff
@@ -1201,6 +1203,9 @@ def next(self):
         else:
             for auxname in self.aux_list:
                 ts = self._auxs[auxname].update_ts(ts)
+            for transform in self.transformation_list:
+                ts = transform(ts, self._transforms[transform])
```

Comment:
This is going to break if the transformation does not take arguments in addition to the time step.